decision/package-config/npm/package.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Silly question: what is this file for, vs. the top-level package.json?

Reply:
The overall project is a typescript project that is used to create the yaml spec. The file you are querying is the package.json for the output package that is saved to an npm registry. They are in effect different projects, and you can think of this file as a template file for the package which is used by the publish workflow. The publish workflow copies over this template, updates the version to the tag given for the github release.

Reply:
Not silly

Reply:
That folder is intended for any output package files needed, e.g. the stub `README.md` file that is also included.

Reply:
Makes sense. Thanks for the explanation!